### DISCUSSION TITLE
직접 제페토 캐릭터의 애니메이션을 만들고싶어요! [치자피즈]

### DISCUSSION THREAD

**Discussion by x4cf** [naverz/zepeto-studio-kor#154] [3. Creator Samples]
제페토에서는 여러분들의 원할한 창작활동을위해 캐릭터파일을 제공합니다.
1. 제페토 홈페이지(https://studio.zepeto.me/guides/creating-your-item-3d-modelers-1) 에 접속하여 creatorBaseSet_zepeto.zip 파일 (다운로드) - 2021.08 updated 파일을 다운로드합니다
2. 다운로드받은 파일을 가능한 툴로 열어 애니메이션 파일을 제작합니다.
3. 유니티에 넣어 Rig에서 "휴머노이드" 설정을 합니다
4. Apply 눌러 생성된 세모난 아이콘의 애니메이션 클립을 이용합니다.

(보기의 예시 애니메이션은 믹사모 애니메이션을 다운로드 받아 사용하였습니다.)

<img width="1446" alt="제페토 모델링 파일" src="https://user-images.githubusercontent.com/90679926/147799750-fd3deb71-3f3b-43d4-a26d-08c449d4cb93.png">
<img width="2560" alt="블랜더에서 열은 파일" src="https://user-images.githubusercontent.com/90679926/147799752-c8a79f4f-8820-4f9c-9aee-16820f1ff2da.png">
<img width="950" alt="휴머노이드" src="https://user-images.githubusercontent.com/90679926/147799753-7895b937-ad0f-4459-b581-f3e7590414ad.png">
<img width="950" alt="클립" src="https://user-images.githubusercontent.com/90679926/147799756-58da677f-8d48-468a-8f7c-3ac34acb9c60.png">

여러분들은 어떻게 개발하고계신가요? 위에 제시된 방법을 제외하고도 좋은 정보가있다면 알려주세요 XD

틀린 정보가 있다면 댓글달아주세요 :D

**Comment by rjc90** [comment]
감사합니다!!!!!!!
